head.html

```diff
@@ -1,4 +1,4 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
-<script src="/scripts/scripts.js" type="module" crossorigin="use-credentials"></script>
+<script src="/scripts/scripts.js" type="module" crossorigin="anonymous"></script>
```

Comment:
This is required to avoid the CORS issue that comes with the dynamic import of the web-vitals module. I'm setting `Access-Control-Allow-Origin: *` when serving the RUM script, but this value is only accepted when no credentials get passed.

The only alternative would be to change the CDN config to serve the module from the same host, but as discussed with @stefan-guggisberg this is something we'd like to avoid.